pbrd/pbr_zebra.h

```diff
@@ -39,4 +39,10 @@ extern void pbr_send_pbr_map(struct pbr_map_sequence *pbrms,
 			     struct pbr_map_interface *pmi, bool install);
 
 extern struct pbr_interface *pbr_if_new(struct interface *ifp);
+
+extern int pbr_ifp_create(struct interface *ifp);
```

Comment:
Can you put these in a `pbr_interface.c` file and then call a `pbr_interface_init()`? Just to be more consistent.

Reply:
Why?  Not all daemons have a xxx_interface.c file nor is it clear to me that we need another file in pbr to hold 4 functions.  What are you hoping to do?

Reply:
I guess I wrongly assumed this was standardized. I'm fine with whatever as long as we are consistent moving forward.

There also exists a `*_vrf.c` file in many of the daemons doing the same thing. Should this be in `*_zebra.c` as well?

Reply:
No each daemon has taken a somewhat different approach here.  I'm fixing a bug here not necessarily trying to standardize behavior.  And yes I realize I go in and poke stuff around a bunch.  Not sure I'm interested in standardizing this across all daemons( which really is the correct long term behavior ).  As that I think this needed to be applied a bunch of places and I didn't want to have to make it work right across them.